### PR TITLE
[CSF-2315] Node 10.8.0

### PR DESCRIPTION
**Pull request by aklkv**

### CODE REVIEW

**Comment by aklkv** [comment]
@mflatscher @avalonabecker this now tested locally and ready for review

File: node-express/config/passport.js [hunk 1]
```diff
@@ -25,7 +25,6 @@ module.exports = function (passport, config) {
 			path: config.passport.saml.path,
 			entryPoint: config.passport.saml.entryPoint,
 			issuer: config.passport.saml.issuer,
```
Comment (avalonabecker) [inline node-express/config/passport.js]: Why was the cert removed here?

Comment (aklkv) [reply node-express/config/passport.js]: That config value is coming in undefined, consequently breaking passport-saml. I have inspect env config and it doesn’t look like this cert is ever being set. I might be wrong.

**Comment by aklkv** [comment]
@avalonabecker I have test this locally multiple times destroying and bringing back up the whole stack.

File: node-express/config/passport.js (hunk 1 above)
Comment (avalonabecker) [reply node-express/config/passport.js]: @mflatscher - I believe you did the setup on this long ago... do you have any recollection of how/if config.passport.saml.cert is used in node-express/config/passport.js, or could be eliminated?
```
	passport.use(new SamlStrategy(
		{
			path: config.passport.saml.path,
			entryPoint: config.passport.saml.entryPoint,
			issuer: config.passport.saml.issuer,
			cert: config.passport.saml.cert
		},
		function (profile, done) {
			return done(null,
				{
					userId: local ? crypto.createHash('md5').update(profile.EmailAddress + profile.FirstName + profile.LastName).digest('hex') : hashUserId(profile.amauniqueid),
					email: local ? profile.EmailAddress : profile.email,
					givenName: local ? profile.FirstName : profile.givenname,
					familyName: local ? profile.LastName : profile.surname,
					displayName: local ? `${profile.FirstName} ${profile.LastName}` : profile.displayname,
					username: local ? profile.EmailAddress : profile.username,
					tamId: local ? "unknown" : profile.amauniqueid,
					groups: local ? "unknown" : profile.memberOf		
				});
		})
```

**Pull request merged**